### PR TITLE
Hit-THE-API

### PR DESCRIPTION
**Pull request by shuja-shah**
### This Branch Implements The following
- User can now add the scores.
- Users can now see the latest scores of our game 😄 

When the user submits the form, all the data is sent to an API, then the API updates the Leaderboard when the user hits the refresh button.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -7,6 +7,26 @@
...
+        </nav>
+    </header>
+    <main>
+        <section id="leaderboard">
+        </section>
+        <section id="addNew">
+            <h2>Add your Score</h2>
+            <form>
+                <input type="text" id="name" name="name" placeholder="Your Name">
+                <input type="text" id="score" name="score" placeholder="Your Score">
```
Comment (zainsadaqat) [inline dist/index.html:26]: - [1] Please validate the form by not allowing the user to submit empty input fields
- [2] Score input field should only accept numbers please replace `type=text` with the type that only accepts numbers. 😃

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @shuja-shah  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shah-shuja-dev** [comment]
### Hello, Thank you for reviewing my project, I've implemented the requested changes. :)

**Review by aimemalaika — APPROVED** [review]
Hi @shuja-shah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**